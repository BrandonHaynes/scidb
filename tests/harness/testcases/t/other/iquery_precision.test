--setup
--start-query-logging

create array foo <x:double> [i=1:10,10,0]
store(build(foo, 1.0/i), foo)

--test

--shell --store --command "iquery -olcsv+ -aq 'scan(foo)'"
--shell --store --command "iquery -olcsv+ -w 10 -aq 'scan(foo)'"

--cleanup
remove(foo)
--stop-query-logging
