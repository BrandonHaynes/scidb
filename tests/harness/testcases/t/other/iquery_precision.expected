SCIDB QUERY : <create array foo <x:double> [i=1:10,10,0]>
Query was executed successfully

SCIDB QUERY : <store(build(foo, 1.0/i), foo)>
{i} x
{1} 1
{2} 0.5
{3} 0.333333
{4} 0.25
{5} 0.2
{6} 0.166667
{7} 0.142857
{8} 0.125
{9} 0.111111
{10} 0.1

SCIDB QUERY : <iquery -olcsv+ -aq 'scan(foo)'>
i,x
1,1
2,0.5
3,0.333333
4,0.25
5,0.2
6,0.166667
7,0.142857
8,0.125
9,0.111111
10,0.1

SCIDB QUERY : <iquery -olcsv+ -w 10 -aq 'scan(foo)'>
i,x
1,1
2,0.5
3,0.3333333333
4,0.25
5,0.2
6,0.1666666667
7,0.1428571429
8,0.125
9,0.1111111111
10,0.1

SCIDB QUERY : <remove(foo)>
Query was executed successfully

